In [1]:
import numpy as np
import pandas as pd
%matplotlib inline

In [2]:
df = pd.DataFrame()
df['age'] = [14,12,11,10,8,6,8]

In [3]:
np.mean(df['age'])

9.857142857142858

In [4]:
np.median(df['age'])

10.0

In [6]:
import statistics
statistics.mode(df['age'])

8

In [18]:
df = pd.DataFrame()

In [19]:
df['age2'] = [20,23,17,5]

In [20]:
np.mean(df['age2'])

16.25

In [21]:
np.median(df['age2'])

18.5

In [15]:
statistics.mode(df['age2'])

StatisticsError: no unique mode; found 7 equally common values

2. Median because it's least affected by skews.
3. Mean changed to 10 (matches the median)
4. No because the median helps with skewness
5. 18.5%